Makefile

```
@@ -26,11 +26,14 @@ K6 ?= $(shell if command -v k6 >/dev/null 2>&1; then command -v k6; elif command
 K6_BASE_URL ?= http://localhost:4433
 K6_RUN_ID ?= $(shell date '+%Y%m%d%H%M%S')
 K6_TEST_PASSWORD ?= K6pass!1234
+K6_TEST_PASSWORD_ENV := $(K6_TEST_PASSWORD)
 FULL_VUS ?= 1
 FULL_ITERATIONS ?= 1
 FULL_MAX_DURATION ?= 5m
 FULL_SLEEP_SECONDS ?= 0
 
+k6-full k6-idor: export K6_TEST_PASSWORD = $(K6_TEST_PASSWORD_ENV)
```

checkmake (0.3.2)

[warning] Target "k6-full k6-idor" should be declared PHONY.

(phonydeclared)